pkg/webhook/mutation.go

```diff
@@ -90,6 +90,9 @@ func AddMutatingWebhook(mgr manager.Manager, deps Dependencies) error {
 	}
 	server := mgr.GetWebhookServer()
 	server.TLSMinVersion = *tlsMinVersion
+	if *DisableCertRotation && *clientCAName != "" {
+		server.ClientCAName = *clientCAName
```

Comment:
We should also add this to the `policy.go` webhook file, in case the user has disabled the mutation webhook (or refactor the initialization pipeline so we set global server configurations in a single place).

Reply:
Should we also add this to `namespacelabel.go` as well?

Reply:
+1 good call